### PR TITLE
Complete translations for Czech locale (cs-CZ)

### PR DESCRIPTION
**Pull request by sinkien**
| Q               | A
| --------------- | ---
| Bug fix?        | [ ]
| New feature?    | [x]
| New sample?      | [ ]
| Related issues?  |  none

#### What's in this Pull Request?

This PR contains a new file cs-cz.ts added to src/loc. This file contains complete translations of strings for Czech locale (cs-CZ).

### CODE REVIEW

File: src/loc/cs-cz.ts [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+    "DynamicFormRequiredErrorMessage": "Toto pole nemůže zůstat prázdné.",
+    "DynamicFormTermPanelTitle": "Vybrat výraz",
+    "DynamicFormEnterURLPlaceholder": "Zadejte URL",
+    "DynamicFormEnterDescriptionPlaceholder": "Alternativní popis",
+    "DynamicFormDialogValidationErrorTitle": "Chyba validace",
+    "DynamicFormDialogValidationErrorMessage": "Formulář obsahuje chyby validace. Před uložením je potřeba tyto chyby odstranit.",
+    "DynamicFormNumberValueMustBeGreaterThan": "Hodnota musí být větší než {0}",
+    "DynamicFormNumberValueMustBeBetween": "Hodnota musí být mezi {0} a {1}",
+    "DynamicFormNumberValueMustBeLowerThan": "Hodnota musí být menší než {0}",
+    "customDisplayName": "Použít toto umístění:",
```
Comment (michaelmaillot) [inline src/loc/cs-cz.ts:386]: There're missing location props that should be translated:

* `DynamicFormChooseFileLabel`
* `DynamicFormChooseFileButtonText`
* `DynamicFormRequiredFileMessage`

You can find example [here](https://github.com/pnp/sp-dev-fx-controls-react/blob/dev/src/loc/en-us.ts#L391)

File: src/loc/cs-cz.ts [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+
+    "ProgressStepsIndicatorNoSteps": "Žádné kroky",
+    "TermSertNaviagtionErrorMessage": "Výraz nebyl nalezen",
+    "TermSetNavigationNoTerms": "Výrazy nejsou definovány",
+
+    "UploadFilesButtonLabel": "Přidat",
+    "UpLoadFilesDeleteButtonLabel": "Odstranit",
+    "UpLoadFilesDragDropLabel": "Přetáhněte soubory sem, nebo klikněte na tlačítko Přidat",
+    "UpLoadFilesSelectAllLabel": "Vybrat vše",
+    "UploadFilesUploadButtonLabel": "Nahrát",
```
Comment (michaelmaillot) [inline src/loc/cs-cz.ts:419]: There's a missing location prop regarding a new control added recently :

* `HoverReactionBarSearchEmojiPlaceholder`

You can find it [here](https://github.com/pnp/sp-dev-fx-controls-react/blob/dev/src/loc/en-us.ts#L429)

File: src/controls/dynamicForm/DynamicForm.tsx [hunk 3]
```diff
@@ -597,8 +597,8 @@ export class DynamicForm extends React.Component<
             hiddenName = response.value;
             termSetId = field.TermSetId;
             anchorId = field.AnchorId;
-            if (item !== null) {
-              item[field.InternalName].forEach((element) => {
+            if (item !== null && item[field.InternalName] !== null && item[field.InternalName].results !== null) {
```
Comment (michaelmaillot) [inline src/controls/dynamicForm/DynamicForm.tsx:600]: This update shouldn't be relevant once your branch wull be updated with latest commits on dev

**Review by michaelmaillot — CHANGES_REQUESTED** [review]
Please update the `supported.localization.json` file to add the new locale.You can find it [here](https://github.com/pnp/sp-dev-fx-controls-react/blob/dev/config/supported.localization.json#L2).

**Comment by michaelmaillot** [comment]
Hi @sinkien,

Thanks for this PR!

I've submitted a tiny review regarding the translation file. Also, could you update your branch to reflect latest updates made on dev one?

Let me know if something's wrong.

**Comment by michaelmaillot** [comment]
Hi @sinkien,

Are you still working on this one? Do you need any help?

**Comment by sinkien** [comment]
Scraping this PR, because everything went haywire. I'll submit new.

**Pull request closed without merging**